contracts/core/VolOracle.sol

```diff
@@ -174,6 +177,14 @@ contract VolOracle is DSMath {
                 .mul(annualizationConstant);
     }
 
+    function getTokenPrice(address pairAddress) public view returns(uint) {
+        IUniswapV2Pair pair = IUniswapV2Pair(pairAddress);
```

Comment:
The current tests are failing because the Uni v3 pair was passed in instead

Reply:
They are passing now but I want to deploy a separate `VolOracleV2.sol` for Uniswap V2 forks.  We can sync about TWAP'ing or going the ChainLink oracle route soon.